src/overlays/actors/ovl_En_Fire_Rock/z_en_fire_rock.c

```diff
@@ -24,7 +30,7 @@ const ActorInit En_Fire_Rock_InitVars = {
     (ActorFunc)EnFireRock_Draw,
 };
 
-static ColliderCylinderInit D_80A12CCC = {
+static ColliderCylinderInit D_80A12CA0 = {
```

Comment:
```suggestion
static ColliderCylinderInit sCylinderInit = {
```
Same with the one at line 53

Reply:
I didn't name these yet because I didn't want to just call them sCylinderInit1 and sCylinderInit2. I wanted to wait until the file was documented so we could name them for their specific uses